nilearn/_utils/cache_mixin.py

```diff
@@ -165,7 +172,7 @@ class CacheMixin(object):
     parameter to self._cache(). See _cache() documentation for details.
     """
 
-    def _cache(self, func, func_memory_level=1, **kwargs):
+    def _cache(self, func, func_memory_level, **kwargs):
```

Comment:
> I just wanted to make it explicit and mandatory. No strong feelings about that.

I'd rather not bother people if they are not thinking about this kind of
problem: it's a valid usecase to want to cache without controling the
granularity.